server/views/pages/recommendations/apRationaleConfirmation.njk

```diff
@@ -2,37 +2,39 @@
 {% set pageTitle = makePageTitle({ pageHeading: pageTitles[page.id], hasErrors: errors }) %}
 {% block content %}
     <div class="govuk-grid-row">
-
         <div class="govuk-grid-column-two-thirds-from-desktop">
             <div class="govuk-panel govuk-panel--confirmation">
-              <h1 class="govuk-panel__title">
-                Recall started
-              </h1>
-              <div class="govuk-panel__body" data-qa="fullName">
-              {{ personOnProbation }}
-              </div>
-              <div class="govuk-panel__body" data-qa="crn">
-                NOMIS number: {{ nomsNumber }}
```

Comment:
Is getting rid of NOMIS number intentional?

Reply:
Yes - I was informed on Friday that it is now the `crn` and not `nomis`